### DISCUSSION TITLE
why you've added pubspec.yalm file in Flutter App?

### DISCUSSION THREAD

**Discussion by peacejr7** [rwema3/SongTubeExactApp#2] [Q&A] [answered]
what is the function or role of pubspec.yaml file?

**Comment by rwema3** [comment, marked as answer]
The pubspec. yaml file is transversal to all apps and packages - it is where we add metadata to our project, stipulate the Dart and Flutter SDK constraints, manage the dependencies and also set Flutter-specific configurations.
